src/components/PostForm.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  Stack,
+  StackDivider,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { BlogPostCategory } from "../types/blogPost";
+
+export interface PostFormType {
+  title: string;
+  score: number;
+  categories: BlogPostCategory[];
+  author: string;
+}
+
+export function PostForm({ author, categories, score, title }: PostFormType) {
+  return (
+    <Card bg={"#EDF2F7"} mb={5} mt={5}>
```

Comment:
Let's use one of the chakra's colors https://chakra-ui.com/docs/styled-system/theme